src/main/java/org/maxgamer/rs/event/EventManager.java

```diff
@@ -1,12 +1,13 @@
 package org.maxgamer.rs.event;
 
+import org.maxgamer.rs.core.Core;
+import org.maxgamer.rs.structure.timings.StopWatch;
+import org.maxgamer.rs.util.log.Log;
+
 import java.lang.reflect.Method;
 import java.util.HashMap;
 import java.util.LinkedList;
 
-import org.maxgamer.rs.core.Core;
-import org.maxgamer.rs.structure.timings.StopWatch;
-
 public class EventManager {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)